watertap/unit_models/electrodialysis_0D.py

```diff
@@ -2294,6 +2348,47 @@ def calculate_scaling_factors(self):
                 )
             iscale.set_scaling_factor(self.pressure_drop_total, sf)
 
+        if hasattr(self, "current_dens_lim_ioa"):
+            if iscale.get_scaling_factor(self.current_dens_lim_ioa) is None:
+                if (
+                    self.config.limiting_current_density_method
+                    == LimitingCurrentDensityMethod.InitialValue
+                ):
+                    sf = self.config.limiting_current_density_data**-1
+                    iscale.set_scaling_factor(self.current_dens_lim_ioa, sf)
+                elif (
+                    self.config.limiting_current_density_method
+                    == LimitingCurrentDensityMethod.Empirical
+                ):
+                    sf = 25**-1 * sum(
```

Comment:
What is the reasoning for 1/25 here? It's pretty mysterious reviewing it (without looking back at the equation and seeing what combination of variables is being approximated at 25), but a lot of scaling routines are.

Reply:
I don't know all scaling method needs to be based on a physical meaning.  Here the number was inspired by an empirical coefficient from a literature.